DependencyInjection/Configuration.php

```diff
@@ -370,7 +370,7 @@ private function addSharedPluginNodes(ArrayNodeDefinition $pluginNode, $disableA
                         ->addDefaultsIfNotSet()
                         ->children()
                             ->scalarNode('default_ttl')->defaultNull()->end()
-                            ->scalarNode('respect_cache_headers')->defaultTrue()->end()
+                            ->arrayNode('respect_response_cache_directives')->defaultNull()->end()
```

Comment:
We should use `scalarNode('respect_cache_headers')->defaultNull()->end()`. That would be the same as "not using it".